ipv8/messaging/anonymization/community.py

```diff
@@ -805,6 +791,12 @@ def on_extend(self, source_address, payload, _):
             if not extend_candidate:
                 extend_candidate = Peer(payload.node_public_key, payload.node_addr)
 
+            # Ensure that we are able to contact this peer
+            if extend_candidate.last_response + 57.5 < time.time():
```

Comment:
Why not use `self.settings.unstable_timeout` instead of `57.5`?

Reply:
Because they are 2 different things. The `57.5` refers to how long after receiving a message from a peer do you consider this peer connectable, whereas the `unstable_timeout` refers to the period during which a circuit is allowed to change.